moped-database/hasura-cluster

```diff
@@ -533,9 +149,9 @@ function start_only() {
   __get_platform;
   START_MODE=$1;
   if [[ "${START_MODE}" = "" ]]; then
-    docker-compose -f docker-compose.yml -f docker-compose.${PLATFORM}.yml up -d;
+    docker compose -f docker-compose.yml -f docker-compose.${PLATFORM}.yml up -d;
   else
-    docker-compose -f "${START_MODE}" up -d;
+    docker compose -f "${START_MODE}" up -d;
```

Comment:
One of the issues in this PR is to transition to using `docker compose` instead of the old way of using the perl script `docker-compose`. The core docker project adopted the compose functionality and brought it into the core package.